php-practice/section6.php

```diff
@@ -223,7 +270,18 @@ function printErrors(array $errors) {
  *
  * Now make the SAME printErrors function with a foreach loop!
  */
-
-function printErrorsAlternate(array $errors) {
-  // YOUR CODE HERE
+/**I'm going to need your help. I got this to work but I'm not sure why I didn't get the "string conversion error" I normally get.
+*/
+function printErrorsAlternate(array $altErrors) {
+  foreach($altErrors as $value) {
```

Comment:
Also, same thing in `foreach` loops -- don't use `$value` unless it makes the most sense. If I were writing this code, I'd probably do:

```PHP
foreach ($errors as $error) {
  // ...
}
```

`$errors` is the list of errors and `$error` is one single error.